fe/fe-core/src/main/java/com/starrocks/authentication/PlainPasswordAuthenticationProvider.java

```diff
@@ -78,15 +80,22 @@ public void authenticate(
             byte[] remotePassword,
             byte[] randomString,
             UserAuthenticationInfo authenticationInfo) throws AuthenticationException {
+        if (randomString != null) {
```

Comment:
What does randomString != null represent?

Reply:
meaning the password sent by mysql client has already been scambled(encrypted) using random string.

Reply:
done